presto-main/src/test/java/com/facebook/presto/sql/planner/TestDomainTranslator.java

```diff
@@ -101,6 +102,7 @@
     private static final Symbol M = new Symbol("m");
     private static final Symbol N = new Symbol("n");
     private static final Symbol O = new Symbol("o");
+    private static final Symbol P = new Symbol("p");
 
     private static final Map<Symbol, Type> TYPES = ImmutableMap.<Symbol, Type>builder()
             .put(A, BIGINT)
```

Comment:
not the scope of this patch. If I remember correctly, there is some @arhimondr PR that already refactors this part of code.